gateway_lambda/index.functional.ts

```diff
@@ -0,0 +1,240 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import * as Sentry from '@sentry/serverless';
+import * as jwt from './jwt';
+import { eventHandler, formatResponse } from './index';
+import config from './config';
+import {
+  PurgeQueueCommand,
+  ReceiveMessageCommand,
+  ReceiveMessageCommandInput,
+} from '@aws-sdk/client-sqs';
+import { sqsClient } from './sqs';
+import { APIGatewayEvent } from 'aws-lambda';
+import { EVENT } from './types';
+
+const sampleApiGatewayEvent: APIGatewayEvent = {
```

Comment:
quite the object 📖 😂

Reply:
best part is that there's more these are just the required ones